### DISCUSSION TITLE
Introduce Yourself

### DISCUSSION THREAD

**Discussion by GingerKiwi** [GingerKiwi/alice-game#4] [General]
This is a project for new devs wanting to learn open source web development and version control (github) with others. The first step is to introduce yourself in the discussions tab. Please answer the following quick questions.

1. What is your real first name and Github profile link?
2. Where in the world are you (country and time zone)?
3. Where are you in your learning journey? -> Just learning html, know html and css, just started learning javascript, comforatble with more advanced javascript such as promises, api, etc.?
4. How are you learning? -> Self taught, bootcamp, computer science courses
5. What would you like to work on in this repo

**Comment by GingerKiwi** [comment]
Hi! I'm the creator and maintainer of this repo.

1. Liz (Elizabeth) - https://github.com/GingerKiwi
2. Toronto, Canada - EDT
3. I love CSS, am comfortable with vanilla javascript, have done some backend, have a Jamstack blog, and are just starting in react. I'm also exploring Go/Golang, Rust, and Typescript.
4. I took a very broad range of computer and information sciences courses in 2020-2021 online at Massey U in New Zealand. I'm now doing the more focused (and more effective) Scrimba Frontend Developer Career Path program. 
5. I'm working on learning how to maintain an open source repo.

**Comment by devils2ndself** [comment]
Hey! I'd like to make a contribution and here's a brief description of who I am.

1. Artem - https://github.com/devils2ndself
2. Toronto, Canada (EST/EDT)
3. I am a full-stack developer, on my path of becoming a successful software engineer. I usually like to work on back-end and system design, but I do sometimes love to code front-end to keep my skills in shape.
4. I am a senior year college student, but development engineering is really all about self-teaching, which is where major part of my knowledge comes from.
5. I'd like to work on a couple of front-end issues

**Reply by GingerKiwi** [reply]
Welcome! Thanks for reading and following the instructions. Are you involved with [Toronto Javascript](https://torontojs.com)? If not, it's a really friendly and supportive community.

**Reply by devils2ndself** [reply]
Interesting! Never heard of it, thank you for sharing!

**Comment by sivasuriyankumarasamy** [comment]
Hey, 

Sivasuriyan - https://github.com/sivasuriyankumarasamy
India (IST)
I am a full-stack developer, But I usually work with backend in Java. I wanted to start learn something in python. 
I learned program by self learning.
I would like to learn some python or front-end.

**Reply by GingerKiwi** [reply]
Hi Sivasuriyan,
Welcome! It's nice to try new things isn't it? I'll assign you the issue.

**Reply by sivasuriyankumarasamy** [reply]
yes it is nice to try/learn new things. Thanks

**Comment by Mirza-Hassan** [comment]
Hey, 

Mirza - https://github.com/mirza-hassan
Pakistan (PST)
I am a full-stack developer, But I usually work with frontend in Javascript. I love open source and community building.

**Reply by GingerKiwi** [reply]
Welcome Mirza!
Where did you learn to code?

**Reply by Mirza-Hassan** [reply]
I learned coding from udemy udacity online course.

**Reply by GingerKiwi** [reply]
Thanks again for being persistent Mirza in learning Markdown and getting that PR done!

**Comment by DHRUVKHANDELWAL00** [comment]
Hey! I'd like to make a contribution and here's a brief description of who I am.

Artem - https://github.com/DHRUVKHANDELWAL00
RAJASTHAN,INDIA
I am a full-stack developer, on my path of becoming a successful software engineer.
I am a 2nd year college student, but development engineering is really all about self-teaching, which is where major part of my knowledge comes from.
I'd like to work on a couple of front-end issues

**Comment by SKVKPandey** [comment]
Hey,

1. My name is Shashank Kashyap Vijay Kumar Pandey and my GitHub profile link is: https://github.com/SKVKPandey
2. South Delhi, Delhi, India.
3. Currently, I work as a freelancer web developer (full-stack) and I love contributing to open-source projects.
4. I am a 2nd Year electronics student and I have learned to code by practicing myself and making projects.
5. I'd like to contribute to the CSS part of the front end.

**Reply by GingerKiwi** [reply]
Hi Shashank, 
Thanks for introducing yourself.
Issue 21 is open. It's a quick html one. 
I'll add some css issues later today (10-12 hrs).
The open css @font issue is being claimed by someone.

**Comment by SKVKPandey** [comment]
I have done it.

**Comment by Vishu-uut** [comment]
Hey, I am Vishwas Gupta from Delhi, India.
I am currently a sophomore at IIT-D pursuing Production and Industrial Engineering.
I want to try my hands at some open-source projects.
I am comfortable with HTML and CSS, but my real strong suit is javascript.
I have learned web development from Colt Steel's Udemy course. Complete Web Development Bootcamp.
https://github.com/Vishu-uut

**Comment by Zien-Alhawshi** [comment]
Hii, nice to meet you all! 
1- I'm Zien Alhawshi this is my github link: https://github.com/Zien-Alhawshi
2- I'm living in Syria 
3- I'm learning front end development/ I'm comfortable with HTML/CSS and I'm working to improve my skills in JS and learn React in the future
4- I'm fresh graduate from computer engineering faculty, but these days my I'm depending on Scrimba front end path as a main resource
5-I would like to work on front end part of the app!

**Comment by michaella23** [comment]
Hello! My name is **Michaella** and I live in Austin, Texas, USA (Central Standard Time, after this weekend). My GitHub user profile is [michaella23](https://github.com/michaella23). I am always learning, but I graduated from Scrimba's frontend developer career path this past spring. I started my coding journey a little over a year ago, and I love working with HTML, CSS, JavaScript and React, with a focus on accessibility. This project seems right up my alley! I am interested to jump in and help wherever I can, mainly the frontend I'm guessing. I've been learning a bit of Java and SQL, but I haven't worked with Python before. I like looking out for good use of semantic HTML, contrast ratios, alt text, and other accessibility issues. I'm still learning best practices as far as that goes. Thanks for starting this repo, and I hope I'll be able to contribute!

**Reply by michaella23** [reply]
Also, I am pretty new to open source myself, but I helped maintain a repository during Hacktoberfest and now feel more comfortable with both sides of the GitHub work flow. I'm certainly still learning and make mistakes just like anyone, but if you end up needing someone to help maintain this repository I would be open to it! 😊

**Reply by GingerKiwi** [reply]
Hi @michaella23 
Yeah! Another Scrimba dev! Awesome to have you. I'm focusing on getting my blog re-design up and getting some content and tech writing contracts the next week or so. 

But I'll be back in here and figuring out a schedule and such soon.

Note: You just have to be able to read python. We're in Javascript land in this rabbit hole. ;-)

**Comment by CBID2** [comment]
Hey everyone,! My name is Christine(she/her), and I live in the U.S. (EST). I started my coding in May after being rejected from an Instructional Design position for not having a basic knowledge of HTML and CSS (I noticed that more postings are making similar qualifications). Right now, I’ve been focusing on using Flexbox more in my projects (I use FrontEnd Mentor, FreecodeCamp, and Scrimba as knowledge gap fillers. If you want to learn more about my journey, check out my [LinkedIn post](https://www.linkedin.com/posts/christinebelzie_coding-workshops-for-women-activity-6932511102119395328-R5s-?utm_source=share&utm_medium=member_ios).). My GitHub profile is [CBID2](https://github.com/CBID2). Even though I have been doing Open Source projects for a while, it’s still a fairly new concept for me, so I would love to be a Contributor this repo to build my coding skills, especially with HTML & CSS. However, if that role ends up being filled, I’m open to being a Maintainer! :) Looking forward to learning from you all! 😊

**Reply by GingerKiwi** [reply]
Hi Christine!
Scrimba representing! 😊
Open source and GitHub workflows are a puzzle I'm enjoying figuring out together.

Like I've mentioned to Michaella just now and on the About page, I'm focusing on my blog and getting some writing gigs (need to pay for bills and kitty cat food!).
I'll be back down the rabbit hole with this project in the next week or so.

I'm so thrilled that this project is gaining momentum!

**Comment by VanjaXYZ** [comment]
1. Vanja, https://github.com/VanjaXYZ
2. Bosnia
3. Intermediate HTML&CSS, Intermediate JS (I've worked with APIs tho), started learning React a couple days ago
4. Self-tought
5. Everything :)

**Comment by rmach1992** [comment]
1. Raymond https://github.com/rmach1992
2. California, PST
3. I know HTML/CSS/JS and I've done some projects with advanced JS but it is something I would need to review.
4. I am doing self-taught via Scrimba.
5. I heard about this on the announcement and since I've mainly been self-taught, I thought this would be a great opportunity to develop my skills in an open source environment. I'm open to working on anything in this repo.

**Comment by arslonga83** [comment]
Hi everyone,

1. My name is Jeff and my github profile is https://github.com/arslonga83
2. I'm in Pacific time zone, USA
3. I've been learning for about 4 months. At this point I'm comfortable with HTML, CSS, and basic Javascript. I've done a few projects with more advanced javascript like APIs but not much.
4. Self-taught - I started working through the Odin Project but recently discovered Scrimba and started working through the FrontEnd Career Path.
5. I haven't done any open source contributing before, so I'm just interested to learn how it all works and would be happy to help out with any parts of the project!

**Comment by GingerKiwi** [comment]
Hi All the new people!
@arslonga83 @rmach1992 @VanjaXYZ @Zien-Alhawshi 
It's awesome to see you all! Welcome to Wonderland! 

I'm working on project management this week so that we're nice and organized. You'll see there's project linked to this repo now. :-) The goal is to be organized so we can do a first release of the basic working game by January (or end of December).

I'm typing a bit slower than usual because I burnt a finger last night. ... I need to set up some more accessibility features on my MacBook today too!

Cheers,
Liz
